app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -218,7 +222,7 @@
                                     :prefix            => "amqp",
                                     :basic_info_needed => true}
         .form-group
-          .col-md-12{"ng-if" => "#{ng_model}" == "emsCommonModel" && "#{ng_model}.emstype == 'openstack' || #{ng_model}.emstype == 'openstack_infra'"}
+          .col-md-12{"ng-if" => "#{ng_model}" == "emsCommonModel" && "#{ng_model}.emstype == 'openstack' || #{ng_model}.emstype == 'openstack_infra' || #{ng_model}.emstype == 'nuage_network'"}
```

Comment:
This prints the wrong message:

![capture2](https://user-images.githubusercontent.com/8102426/30847277-349bcf20-a29b-11e7-93dd-03e7e4fb7f11.PNG)

May I suggest to add additional `.col-md-12` here for Nuage provider? Similarly as there is one for OpenStack in [L225](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/views/layouts/angular/_multi_auth_credentials.html.haml#L221-L226) and one for VMWare in [L231](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/views/layouts/angular/_multi_auth_credentials.html.haml#L227-L232). I think you will need something more similar to VMWare than OpenStack.

Alternatively, you could make VMWare's message little more general and then hook Nuage there.

Reply:
I completely missed this message there - thanks for spotting!